adapter-base/src/main/java/org/eclipse/hono/adapter/AbstractProtocolAdapterApplication.java

```diff
@@ -525,11 +525,11 @@ protected HonoConnection commandConsumerConnection() {
      * @param commandRouterClient The client for accessing the Command Router service.
      * @return The factory.
      */
-    protected CommandRouterCommandConsumerFactory commandConsumerFactory(final CommandRouterClient commandRouterClient) {
+    protected ProtocolAdapterCommandConsumerFactoryImpl commandConsumerFactory(final CommandRouterClient commandRouterClient) {
 
-        LOG.debug("using Command Router service client, configuring CommandConsumerFactory [{}]",
-                CommandRouterCommandConsumerFactory.class.getName());
-        return new CommandRouterCommandConsumerFactory(commandRouterClient, getComponentName());
+        LOG.debug("using Command Router service client, configuring CommandConsumerFactory [commandRouterClient: {}]",
+                ProtocolAdapterCommandConsumerFactoryImpl.class.getName());
```

Comment:
`commandRouterClient: ` doesn't fit the used value here.

Reply:
fixed